src/python/pants/backend/python/target_types.py

```diff
@@ -465,29 +463,31 @@ def format_invalid_requirement_string_error(
     )
 
 
-class PythonRequirementsField(PrimitiveField):
+class PythonRequirementsField(PrimitiveField[Tuple[Requirement, ...]]):
```

Comment:
To confirm, it's fine to leave this change off, right? It only means that MyPy won't have as precise of info. I want to ensure that plugin authors won't have anything broken.

Reply:
It should not change anything for anyone at runtime. It could however cause mypy checking of plugins written against this to break, but not at runtime. Is your concern about runtime breakage or type-checking?

Reply:
I have modified the `Field` docstring to read:
> If you are directly subclassing `Field`, you will want to give it a type parameter to set the
>    type of `value`.